online-fishing-store/src/assets/scss/_variables.scss

```diff
@@ -1,3 +1,7 @@
 $flexStart: flex-start;
+$flexEnd: flex-end;
 $spaceBetween: space-between;
+$spaceAround: space-around;
 $center: center;
+$screen-medium: 767px;
```

Comment:
Could you divorse these variables in blocks

/**  Flex values */
$flexStart: flex-start;
$flexEnd: flex-end;
$spaceBetween: space-between;
$spaceAround: space-around;
$center: center;

/** Screen sizes */
$screen-medium: 767px;
$screen-larger: 1019px;

And use please common sizes of screen(580px, 768, 991 and etc...)